Formula/linux-pam.rb

```diff
@@ -0,0 +1,40 @@
+class LinuxPam < Formula
+  desc "Pluggable Authentication Modules for Linux"
+  homepage "http://www.linux-pam.org"
+  url "https://github.com/linux-pam/linux-pam/releases/download/v1.5.1/Linux-PAM-1.5.1.tar.xz"
+  sha256 "201d40730b1135b1b3cdea09f2c28ac634d73181ccd0172ceddee3649c5792fc"
+  license any_of: ["BSD-3-Clause", "GPL-1.0-only"]
+  head "https://github.com/linux-pam/linux-pam.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "berkeley-db"
+  depends_on "libprelude"
+  depends_on "libtirpc"
+  depends_on :linux
+
+  skip_clean :la, "etc"
+
+  def install
+    args = %W[
+      --disable-debug
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --disable-selinux
+      --prefix=#{prefix}
+      --includedir=#{include}/security
+      --oldincludedir=#{include}
+      --enable-securedir=#{lib}/security
+      --sysconfdir=#{etc}
+      --with-xml-catalog=#{etc}/xml/catalog
+      --with-libprelude-prefix=#{Formula["libprelude"].opt_prefix}
+    ]
+
+    system "./configure", *args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    assert_match "Usage", shell_output("#{sbin}/mkhomedir_helper 2>&1", 14)
```

Comment:
Can this match a little more? The version? CC @bo98 just FYI that this is a case we may want to add a nicer DSL/audit for (redirecting stderr to stdout) if we don't already.

Reply:
Yeah we should probably look at making `shell_output` and `pipe_output` smarter like `Utils.popen*`. The former is just a basic interpolation into backticks at the moment.

Reply:
Relatedly: not having `shell_output`, `pipe_output`, `system_command`, `Utils.open_read`, etc. would also be nice 😁. Would ideally have a subset that we recommend based on how we expect them to be used (for both Homebrew/brew and formulae although they are likely to be different).

Reply:
Done